src/type_object.rs

```diff
@@ -232,3 +229,25 @@ pub(crate) unsafe fn get_tp_free(tp: *mut ffi::PyTypeObject) -> ffi::freefunc {
         std::mem::transmute(ptr)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    #[allow(deprecated)]
```

Comment:
If this line is commented out and `<PyList as crate::PyTypeInfo>` replaced with just `PyList`, this is the compiler output:

```
warning: use of deprecated trait `type_object::PyTypeObject`: PyTypeObject::type_object was moved to PyTypeInfo::type_object
   --> src/type_object.rs:240:20
    |
240 |         use super::PyTypeObject;
    |                    ^^^^^^^^^^^^
    |
    = note: `#[warn(deprecated)]` on by default

warning: use of deprecated trait `type_object::PyTypeObject`: PyTypeObject::type_object was moved to PyTypeInfo::type_object
   --> src/type_object.rs:243:31
    |
243 |         fn get_type_object<T: PyTypeObject>(py: Python<'_>) -> &PyType {
    |                               ^^^^^^^^^^^^

error[E0599]: no function or associated item named `type_object` found for struct `PyList` in the current scope
   --> src/type_object.rs:249:58
    |
249 |                 get_type_object::<PyList>(py).is(PyList::type_object(py)))
    |                                                          ^^^^^^^^^^^ function or associated item not found in `PyList`
    |
   ::: src/types/list.rs:18:1
    |
18  | pub struct PyList(PyAny);
    | ------------------------- function or associated item `type_object` not found for this
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
    |
237 |     use crate::type_object::PyTypeInfo;
    |
```

... hopefully this is enough for users to figure out how to fix.